.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -746,34 +746,34 @@ for ((i=0;i<$NUM_UNTRUSTED;i++));
 do
     myuser=`printf "untrusted%02d" $i`
     mydir=`printf "/var/local/submitty/autograding_tmp/untrusted%02d" $i`
```

Comment:
Considering using `$(...)` notation?

```suggestion
    myuser=$(printf "untrusted%02d" $i)
    mydir=$(printf "/var/local/submitty/autograding_tmp/untrusted%02d" $i)
```